modules/EvseManager/Charger.hpp

```diff
@@ -191,6 +198,12 @@ class Charger {
     void set_hlc_charging_active();
     void set_hlc_allow_close_contactor(bool on);
 
+    std::optional<std::string> getOcmfData() {
```

Comment:
Add documentation, for example it is surprising that this is not idempotent to me.

Also I am not a fan of having some of the implementation in the header and some in the .cc file. Please put the implementation in the .cc file too, so that all logic is together.